vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -3054,12 +3067,12 @@ EXPORT(int, sceGxmPrecomputedVertexStateSetAllUniformBuffers, SceGxmPrecomputedV
     if (!precomputedState || !precomputedState->uniform_buffers || !bufferDataArray)
         return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
 
-    UniformBuffers *uniform_buffers = precomputedState->uniform_buffers.get(emuenv.mem);
+    UniformBuffer *uniform_buffers = precomputedState->uniform_buffers.get(emuenv.mem);
     if (!uniform_buffers)
         return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
 
-    for (auto b = 0; b < SCE_GXM_MAX_UNIFORM_BUFFERS; b++)
-        (*uniform_buffers)[b] = bufferDataArray[b];
+    for (int i = 0; i + 1 < precomputedState->buffer_count; i++)
+        uniform_buffers[i + SCE_GXM_UNIFORM_BUFFER_OFFSET] = bufferDataArray[i];
```

Comment:
`i+1` and `i + SCE_GXM_UNIFORM_BUFFER_OFFSET` maybe in previous line also should be `i + SCE_GXM_UNIFORM_BUFFER_OFFSET`